nginx/run-nginx

```diff
@@ -28,6 +28,10 @@ preflight_nginx() {
         yes | cp -rf ${NGINX_CONFIG_DIRECTORY}/hydroshare-nginx.conf.template ${NGINX_CONFIG_DIRECTORY}/hs-nginx.conf
         sed -i 's/FQDN_OR_IP/'${FQDN_OR_IP}'/g' ${NGINX_CONFIG_DIRECTORY}/hs-nginx.conf;
     fi
+
+    # Copy custom error page for 503
+    cp server_error_on.html ../hydroshare/static/server_error_on.html
```

Comment:
Why does this file need to be copied in?  Shouldn't we always respond with this template unless the maintenance page is up?

Reply:
cp because we want it in **hydroshare/static** and nginx isn't a django app so I wasn't sure how to get this **server_error_on.html** into that **hydroshare/static** dir without using collectstatic